edx2bigquery/auth.py

```diff
@@ -57,7 +57,9 @@ def get_creds(verbose=False):
       print "service_acct=%s, key_file=%s" % (SERVICE_ACCT, KEY_FILE)
     return get_service_acct_creds(SERVICE_ACCT, KEY_FILE)
   elif KEY_FILE=='USE_GCLOUD_AUTH':
-    return get_gcloud_oauth2_creds()
```

Comment:
Instead of overwriting what is done for `USE_GCLOUD_AUTH`, could you please make this a different option, e.g. `USE_GOOGLE_CREDENTIALS`?